### DISCUSSION TITLE
setAuthorization

### DISCUSSION THREAD

**Discussion by mylaimani** [pedroSG94/RootEncoder#1530] [General]
previously we had 

    rtmpCamera2.setAuthorization(user, password); 

But now we don't have authorization method, is there any other way to do authorization.

**Comment by pedroSG94** [comment]
Hello,

setAuthorization was moved into StreamClient class. This is the equivalent:
```java
rtmpCamera2.getStreamClient().setAuthorization(user, password);
```

**Reply by mylaimani** [reply]
Thank you.
